### DISCUSSION TITLE
Is Nagle's Algorithm disabled?

### DISCUSSION THREAD

**Discussion by andyreimann** [uNetworking/uWebSockets.js#866] [Q&A] [answered]
Hello, we would like to know if Nagle's Algorithm is disabled in uWebSockets and/or if there is a way to control it?

Thank you,
Andy Reimann

**Comment by uNetworkingAB** [comment, marked as answer]
It's disabled, you can't change it. It makes no sense for short message sending like WebSockets and we do our own corking of outgoing data
